src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -149,4 +320,46 @@ private RTCPTransformer()
             super(RTCPPacketPredicate.INSTANCE);
         }
     }
+
+    /**
+     * Adds a {@link MediaStream} to the list of {@link MediaStream}s which
+     * use this {@link TransportCCEngine}.
+     * @param mediaStream the stream to add.
+     */
+    public void addMediaStream(MediaStream mediaStream)
+    {
+        synchronized (mediaStreams)
+        {
+            mediaStreams.add(mediaStream);
+        }
+    }
+
+    /**
+     * Removes a {@link MediaStream} from the list of {@link MediaStream}s which
+     * use this {@link TransportCCEngine}.
+     * @param mediaStream the stream to remove.
+     */
+    public void removeMediaStream(MediaStream mediaStream)
+    {
+        synchronized (mediaStreams)
+        {
+            while(mediaStreams.remove(mediaStream))
+            {
+                // remove all
+            }
+        }
+    }
+
+    /**
+     * @return one of the {@link MediaStream} instances which use this
+     * {@link TransportCCEngine}, or null.
+     */
+    private MediaStream getMediaStream()
```

Comment:
i'm curious...what does the use case end up being where jut getting the first one is sufficient?